requirements.txt

```diff
@@ -35,56 +45,69 @@ hypothesis==4.37.0
 idna==2.8
 imagesize==1.2.0
 importlib-metadata==0.19
-Jinja2==2.11.3
+iniconfig==1.1.1
```

Comment:
I used pip-tools to ensure this file had consistent dependencies. Most of these versions should remain unchanged, but there are some formatting differences. I had to change the versions of pkg-config, pytest, pytest-subtests to be compatible with pytest-regtest. I think the tests cover those dependencies pretty well though.